users/policy.go

```diff
@@ -0,0 +1,93 @@
+package users
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/topicai/candy"
+)
+
```

Comment:
It seems like `Policy` does not need to be an interface. I add receiver `Exists`, `Append`, `Update` for `Policy`